README.md

```diff
@@ -33,7 +33,7 @@ Set the tracker update interval. Disable tracking by setting interval to 0S.
 
 Name | Description | Example
 -- | -- | --
-`update_interval` | `The update interval. Possible values are 0S 30S 1M 5M 10M 30M 1H.` | `30M`
+`update_interval` | `The GPS update interval. Possible values are 0S, 30S, 1M, 2M, 3M, 5M, 10M, 30M, 60M. 0S disables tracking. Battery life will be low for 30S and 1M, high for 2M and 3M and medium for 5M and 10M. Please note, that 30M and 60M are not officially supported in the Weenect app.` | `10M`

```

Comment:
I would remove the battery lifetime info here. I don't see a lot of value in prodiving the information here and also find it confusing, that a higher interval has a lower battery lifetime.

Reply:
I was confused as well. Their help pages point out that when moving there's a turning point where longer intervals will (counter-intuitively) cause higher battery usage. That is because GPS triangulation is quick when your position changes slightly but takes much longer when you've moved significantly. If stationary situations the longer the interval the longer battery life will be. Unfortunately I can not find that article anymore.

Reply:
Okay. We can also leave it in there. No real harm done.